erts/emulator/beam/bif.c

```diff
@@ -54,7 +54,7 @@ static Export* flush_monitor_messages_trap = NULL;
 static Export* set_cpu_topology_trap = NULL;
 static Export* await_port_send_result_trap = NULL;
 Export* erts_format_cpu_topology_trap = NULL;
-static Export dsend_continue_trap_export;
+Export dsend_continue_trap_export;
```

Comment:
If using a `F_FRAGMENTED_SEND` flag would this still need to be exported or can it remain `static`?